tools/reviewer/web/src/app/dashboard/reviews-panel/review/person-list/person-list.component.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  Diff,
+  FirebaseService,
+  NotificationService,
+  ProtoService,
+  Status
+} from '@/shared';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-person-list',
+  templateUrl: './person-list.component.html',
+  styleUrls: ['./person-list.component.scss']
+})
+export class PersonListComponent {
+  @Input() diff: Diff;
+  @Input() property: string;
+  @Input() addToAttention: boolean = true;
+  @Output() onUpdateDiff = new EventEmitter<Diff>();
+
+  // Fields can not be edited if status is
+  // 'SUBMITTED' or 'REVERTED'
+  editable: boolean = true;
+  notEditableStatus: Array<number> = [Status.SUBMITTED, Status.REVERTED];
+
+  // Following variables are used to show editable fields
```

Comment:
Is this to show the edit button?

Reply:
Yeah, `editable` is used to show the little round edit icon.